Hahnium.Chip8/Chip8Ppu.cs

```diff
@@ -49,14 +50,14 @@ public Chip8Ppu(Chip8Cpu cpu, Memory<byte> ram)
         internal void Cycle()
         {
             // Test animation
-            for (int y = 0; y < DisplayHeight; y++)
-            {
-                for (int x = 0; x < DisplayWidth; x++)
-                {
-                    int offset = (y * DisplayWidth) + x;
-                    frameBuffer[offset] = (byte)(((offset + cycleCount + y) % 8) == 1 ? 1 : 0);
-                }
-            }
+            //for (int y = 0; y < DisplayHeight; y++)
```

Comment:
Remove unused code block.

Reply:
Mite move it to it's own function. It's handy enough to keep around for debug/testing.